### ISSUE TITLE
1.11.0-1.3.5 (1.11.2 b 2226) client crashes when pulling crafting furnace from grid

### ISSUE THREAD

**Issue by kreezxil** [ShetiPhian/Localizations-and-Issues#37]
client crash report: https://gist.github.com/kreezxil/5221f933d62b13aa3f05cc86d97fe272

fml client log: https://gist.github.com/kreezxil/e1a49f6219f6669720ad5d487dc2d32e

**Comment by kreezxil** [comment]
apparently this was caused by ProsperCraft removing Terraqueous from the server ... which they did because it interferes with Recurrent Complex during world gen causing server side thread-lock (ie no crash reports and requiring kill -9)

see:

**Comment by ShetiPhian** [comment]
@kreezxil you didn't provide a link.

Since I was never informed of the issue was it found to be in Recurrent Complex?
My generators can be individually disabled, if they can find out which one causes the issue I might be able to find a work around to this conflict.

**Comment by ShetiPhian** [comment]
@kreezxil 
I figure this is the link you intended to include: https://github.com/Ivorforce/RecurrentComplex/issues/155

I've looked over the code Terraqueous and Recurrent Complex seem to conflict over am I believe I've located the issue.

**TL;DR** its cause by a drifting BlockPos in generateSticksStonesAndBones
~~~~
Brief rundown:
1) [Terraqueous] generateSticksStonesAndBones starts a four pass loop
2) [Terraqueous] TerrainGen.decorate is called to see if generation of that block should occur
3) [Forge] fires a Decorate event
4) [Recurrent] uses the event to generate it own stuff and possibly cancel the event
5) [Terraqueous] if the event wasn't canceled the block generates
6) [Terraqueous] generateSticksStonesAndBones starts the next pass, 2-6 repeats

The error happens after the first pass, due to the original BlockPos not getting reset, thus causing TerrainGen.decorate to be given a position that is not equal to 0/0 in the chunk.
When Recurrent's generator adds a random 0-15 the resulting position can be outside of the generating chunk, and an turn causes another chunk to start generating before the first is done.
~~~~

@ProsperCraft
Terraqueous 1.11.0-1.3.6 is now on Curse.
I loaded a new world with RecurrentComplex 1.2.7 with no config changes. Everything appears to be generating fine.
